tracee-ebpf/tracee/net_events.go

```diff
@@ -175,15 +141,19 @@ func (t *Tracee) processNetEvents() {
 	}
 }
 
-func (t *Tracee) writePacket(packetLen uint32, timeStamp time.Time, interfaceIndex int, dataBuff *bytes.Buffer) error {
+func (t *Tracee) writePacket(capData CaptureData, timeStamp time.Time, dataBuff *bytes.Buffer) error {
```

Comment:
I think it will be more efficient to use struct because it firstly reduce the number of args in the function and also  in the next PR ill add a parsing to that struct by `binary.read`.
let me know what you think